### PR TITLE
Rename TraceAttribute to Baggage

### PR DESCRIPTION
**Pull request by bg451**
Per opentracing/opentracing.github.io#67

### CODE REVIEW

File: propagation.go [hunk 1]
```diff
@@ -102,8 +102,8 @@ type SplitTextCarrier struct {
 	// trace.
 	TracerState map[string]string
 
-	// Any Trace Attributes for the encoded Span (per Span.SetTraceAttribute).
-	TraceAttributes map[string]string
+	// Any Baggae for the encoded Span (per Span.SetBaggageItem).
+	TraceBaggage map[string]string
```
Comment (bensigelman) [inline propagation.go]: `s/TraceBaggage/Baggage/g`

File: propagation.go [hunk 2]
```diff
@@ -102,8 +102,8 @@ type SplitTextCarrier struct {
 	// trace.
 	TracerState map[string]string
 
-	// Any Trace Attributes for the encoded Span (per Span.SetTraceAttribute).
-	TraceAttributes map[string]string
+	// Any Baggae for the encoded Span (per Span.SetBaggageItem).
```
Comment (bensigelman) [inline propagation.go]: "Baggae"

File: propagation.go [hunk 3]
```diff
@@ -126,8 +126,8 @@ type SplitBinaryCarrier struct {
 	// trace.
 	TracerState []byte
 
-	// Any Trace Attributes for the encoded Span (per Span.SetTraceAttribute).
-	TraceAttributes []byte
+	// Any Baggage for the encoded Span (per Span.SetBaggageItem).
+	TraceBaggage []byte
```
Comment (bensigelman) [inline propagation.go]: (same)

File: propagation.go (hunk 2 above)
Comment (bg451) [inline propagation.go]: Good catch 👀

**Comment by bensigelman** [comment]
@bg451 basically LGTM... did you verify that `grep -i -r "attr" .` has no other hits? (I.e., no errors of omission here)

**Comment by bg451** [comment]
Yup, double checked right now and all mentions of trace attributes are gone.

**Comment by bensigelman** [comment]
@bg451 the only annoying thing here is the circular dependency that is breaking the Travis CI... `dapperish` proper could move to `basictracer-go/examples` and the `opentracing-go` tests could depend on some minimal extension of the Noop tracer.

**Comment by bg451** [comment]
ping @bensigelman. Rebased the branch and now everything works

**Comment by bensigelman** [comment]
I just noticed that :)

Thanks, LGTM and merging!

**Pull request merged**